server/index.js

```diff
@@ -12,8 +14,9 @@ if (process.env.NODE_ENV === 'development') {
 }
 
 app.use(express.static(publicPath));
+app.use(staticMiddleware);
```

Comment:
This is actually the static middleware registered twice 😂 You can remove line 16